symfony/framework-bundle/4.2/src/bootstrap.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Dotenv\Dotenv;
+
+$envFromEnv = $_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? null;
+$env = $envFromEnv ?? 'dev';
+$prod = 'prod' === $env;
+
+if (!$prod && class_exists(Dotenv::class)) {
+    (new Dotenv())->loadForEnv($env, __DIR__.'/../.env');
```

Comment:
Should we add `$env = $_SERVER['APP_ENV'] ?? $env` after this line, so https://github.com/symfony/recipes/pull/466#discussion_r220616063 still works? I.e changing `APP_ENV` in .env (and no APP_ENV env var provided) would still allow running in a different env?

Reply:
Good idea, I also factorized most of the code in `console` and `index.php`